libs/vm_modules/src/ml/utilities/scaler.cpp

```diff
@@ -79,11 +86,27 @@ void VMScaler::Bind(Module &module)
 {
   module.CreateClassType<VMScaler>("Scaler")
       .CreateConstructor(VMScaler::Constructor)
-      .CreateMemberFunction("setScale", &VMScaler::SetScale)
+      .CreateMemberFunction("setScale", &VMScaler::SetScaleByData)
+      .CreateMemberFunction("setScale", &VMScaler::SetScaleByRange)
       .CreateMemberFunction("normalise", &VMScaler::Normalise)
       .CreateMemberFunction("deNormalise", &VMScaler::DeNormalise);
 }
 
+bool VMScaler::SerializeTo(serializers::MsgPackSerializer &buffer)
+{
+  buffer << *this;
+  return true;
+}
+
+bool VMScaler::DeserializeFrom(serializers::MsgPackSerializer &buffer)
+{
+  buffer.seek(0);
```

Comment:
Is seek(0) correct here? There might have been previous objects serialised in the buffer.